templates/sqoop-parquet-hdfs-hive-merge/create-merge-view.sql

```diff
@@ -0,0 +1,30 @@
+{#  Copyright 2017 Cargill Incorporated
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. #}
+
+set sync_ddl=1;
+USE {{ conf.staging_database.name }};
+
+--Create merged view --
+DROP VIEW IF EXISTS {{ table.destination.name }}_view;
```

Comment:
Instead of calling this just `view`, maybe call it `reconciliation_view` like the blog post? Would help readers associate it with the logic in the post

Reply:
Agree